### PR TITLE
Add Symfony security package

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: config/bundles.php [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <?php
 
-declare(strict_types=1);
```
Comment (kissifrot) [inline config/bundles.php]: Attention :)

File: config/services.yaml [hunk 2]
```diff
@@ -45,3 +45,7 @@ services:
     App\Form\Type\UserType:
         arguments:
             $availableSkillSets: '%app.available_skill_sets%'
+
+    App\Security\LoginFormAuthenticator:
+        arguments:
+            $password: 'covid19'
```
Comment (mtarld) [inline config/services.yaml]: Newline :)

File: src/Controller/Security/LoginController.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+final class LoginController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function __invoke(AuthenticationUtils $authenticationUtils): Response
+    {
+        // if ($this->getUser()) {
```
Comment (mtarld) [inline src/Controller/Security/LoginController.php]: Could be deleted

File: config/packages/security.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            # activate different ways to authenticate
+            # https://symfony.com/doc/current/security.html#firewalls-authentication
+
+            # https://symfony.com/doc/current/security/impersonating_user.html
+            # switch_user: true
+
+    # Easy way to control access for large sections of your site
+    # Note: Only the *first* access control that matches will be used
+    access_control:
+        # - { path: ^/admin, roles: ROLE_ADMIN }
```
Comment (mtarld) [inline config/packages/security.yaml]: We could maybe add paths that are secured (ROLE_USER) now ?

Comment (hhamon) [reply config/packages/security.yaml]: Yes! What are they?

File: src/Controller/Security/LoginController.php (hunk 3 above)
Comment (hhamon) [reply src/Controller/Security/LoginController.php]: Fixed

File: config/services.yaml (hunk 2 above)
Comment (hhamon) [reply config/services.yaml]: Fixed

File: config/packages/security.yaml (hunk 4 above)
Comment (kissifrot) [reply config/packages/security.yaml]: All except ^/user/new and the login URL should be secured I think

File: config/bundles.php (hunk 1 above)
Comment (hhamon) [reply config/bundles.php]: Fixed

File: config/packages/security.yaml (hunk 4 above)
Comment (hhamon) [reply config/packages/security.yaml]: :+1:

Comment (hhamon) [reply config/packages/security.yaml]: Everything is fixed! All good!

**Pull request merged**